tests/chez/chez030/run

```diff
@@ -0,0 +1,4 @@
+$1 --directive extraRuntime=extensions.scm -o chez030 ExtraRuntime.idr
```

Comment:
Test must be run with the additional `--no-color --console-width 0` flags, in this particular case it may not matter, but without these flags compiler output may be dependent on the particular system, due to color and automatic detection of console width. (I should really find a place where to put this indication, sorry)

Reply:
No problem.  I don't think there's any output in this case but I added those options anyway just to be on the safe side.